zoo/src/main/scala/com/intel/analytics/zoo/serving/http/FrontEndApp.scala

```
@@ -57,6 +57,41 @@ object FrontEndApp extends Supportive with EncryptSupportive {
         }
       }
 
+      val servableManager = new ServableManager
+      timing("load servable manager")() {
+        try servableManager.load(arguments.servableManagerPath)
+        catch {
+          case e: ServableLoadException =>
+            logger.info("ServableLoadException: " + e.message)
+            return
+          case e =>
+            logger.info("Loading Config File Error: " + e)
+            val exampleYaml =
+              """
+                ---
+                 modelMetaDataList:
+                 - !<ClusterServingMetaData>
+                    modelName: "1"
+                    modelVersion:"1.0"
+                    redisHost: "localhost"
+                    redisPort: "6381"
+                    redisInputQueue: "serving_stream2"
+                    redisOutputQueue: "cluster-serving_serving_stream2:"
+                 - !<InflerenceModelMetaData>
+                    modelName: "1"
+                    modelVersion:"1.0"
+                    modelPath:"/"
+                    modelType:"OpenVINO"
+                    features:
+                      - "a"
+                      - "b"
+              """
+            logger.info("Example Format of Input:" + exampleYaml)
+            return
```

better to throw this exception